views/js/pciCreator/dev/mathEntryInteraction/pciCreator.json

```diff
@@ -0,0 +1,53 @@
+{
+    "typeIdentifier": "mathEntryInteraction",
+    "label": "Math entry",
+    "short": "Math entry",
+    "description": "Allow test taker to type math formulas",
+    "version": "0.1",
```

Comment:
0.1.0 would be better, I not sure of how the version comparing function will behave with the missing figure.